packages/core/admin/ee/admin/pages/SettingsPage/pages/AuditLogs/ListView/index.js

```diff
@@ -28,14 +28,22 @@ import useAuditLogsData from './hooks/useAuditLogsData';
 const ListView = () => {
   const { formatMessage } = useIntl();
   const {
-    allowedActions: { canRead },
+    allowedActions: { canRead: canReadAuditLogs },
   } = useRBAC(adminPermissions.settings.auditLogs);
+
+  const {
+    allowedActions: { canRead: canReadUsers },
+  } = useRBAC(adminPermissions.settings.users);
```

Comment:
Could we combine these 2 hook calls by just giving it `adminPermissions.settings`? Or do you think it would be worse for performance?

Reply:
I saw SSO is handling something similar like [this](https://github.com/strapi/strapi/blob/06e7dfbd20fafddd26f9e0633c7cc48cc257ad2c/packages/core/admin/ee/admin/pages/SettingsPage/pages/SingleSignOn/index.js#L31-L35) so I did the same.